### PR TITLE
[JS_HW][JS_1]

### PR DESCRIPTION
**Pull request by JackBlaaack**
Homework with variables

### CODE REVIEW

File: task3.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// Task3**. Напишите программу, которая принимает целое положительное число n (одно любое число от 1 до 9), и выводит сумму равную
+
+// n + nn + nnn, где n не перемножаются, а конкатенируются
+
+let n = 9;
```
Comment (anatoly-karpovich) [inline task3.js:5]: Use const

File: task3.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+let result = n + +(n + '' + n) + +(n +'' + n + n )
```
Comment (anatoly-karpovich) [inline task3.js:6]: Use const

**Review by anatoly-karpovich — APPROVED** [review]

File: task3.js (hunk 1 above)
Comment (JackBlaaack) [reply task3.js:5]: I see)

File: task3.js (hunk 2 above)
Comment (JackBlaaack) [reply task3.js:6]: thanks for advice)

**Pull request merged**